src/servers/Cargo.toml

```
@@ -86,7 +86,7 @@ opensrv-mysql = { git = "https://github.com/datafuselabs/opensrv", rev = "a1fb4d
 opentelemetry-proto.workspace = true
 otel-arrow-rust.workspace = true
 parking_lot.workspace = true
-pgwire = { version = "0.28.0", default-features = false, features = ["server-api-ring"] }
+pgwire = { version = "0.29.0", default-features = false, features = ["server-api-ring"] }
```

According to our code style, we don't specify the minor version of crates
```suggestion
pgwire = { version = "0.29", default-features = false, features = ["server-api-ring"] }
```